app/assets/stylesheets/components/forms.scss

```diff
@@ -168,6 +168,10 @@
     font-size: var(--fs-s);
     font-weight: var(--fw-normal); // for when it's inside --label
     margin: 0; //todo remove
+
+    &:empty {
+      display: none;
+    }
```

Comment:
I've seen where/why you used it but I'm wondering why not just drop the code that should technically display something but doesn't do so..?